CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ All notable changes to this project will be documented in this file.
 -   #1605 : Add support for methods and interfaces in classes (including `__init__` and `__del__`).
 -   #1618 : Add support for class instance attributes.
 -   #1680 : Add support for `typing.Final`.
+-   #1529 : Add Python support for list method `append()`
```

Comment:
Please move this to the new UNRELEASED section

Reply:
Done